src/migrations/1727363068702-calloutTemplatesVisibility.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CalloutTemplatesVisibility1727363068702
+  implements MigrationInterface
+{
+  // Bugfix Server#4568
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      ALTER TABLE \`callout\` ADD COLUMN \`isTemplate\` BOOLEAN NOT NULL DEFAULT FALSE AFTER \`type\`;
```

Comment:
That doesn't seem generated by typeorm and will create discrepancies. You don't have a decorator setting the default value in the entity definition.